website/docs/reference/integrations/jira-cloud-plugin-installation.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Jira Cloud Integration - Installation
+---
+
+## Prerequisites
+
+### Jira Cloud
+
+- [Jira **Cloud**](https://www.atlassian.com/software/jira)
+
+For Jira Data Center, check out the [Jira Server plugin](jira-server-plugin-installation.md)
+
+### Unleash
+
+- Unleash v4 or higher
+
+## Required access levels
+
+### Unleash
+
+You will need an Unleash admin user to configure the access tokens needed to connect the plugin to Unleash.
+
+For Unleash Open Source this plugin requires the use of an [Admin API token](../api-tokens-and-client-keys.mdx) to connect.
+
+For Unleash Enterprise we recommend using a [service account](../service-accounts.md). Service accounts are also required to integrate with [Change Requests](../change-requests.md)
+
+### Jira
+
+You will need a Jira admin user.
+
+## Installation
+
+If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+
+> More to be added here after we can have screenshots and guides of the installation procedure
+
+## Configuring the plugin {#configure-plugin}
+
+After the plugin has been installed, each project's settings page in Jira will have a menu entry link called "Unleash Project Settings" under the "Apps" menu section.
+
+![Jira Cloud - Link to Unleash Project Settings in Jira Project Settings Apps menu](/img/jira-cloud-project-settings-link.png)
```

Comment:
Hmm, it feels like the main point of the image is to point to the project settings link, so let's see if we can do the same in text (without repeating the preceding paragraph).

So for a regular image:

```suggestion
![Jira Cloud project settings page with the apps menu open. The link to Unleash project settings is highlighted.](/img/jira-cloud-project-settings-link.png)
```

Reply:
Thank you, looks like a good improvement!